_component_design/text-inputs.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Text Inputs
+title: Text Fields
```

Comment:
We should rename the file as well to standardize so the Text Fields title is replicated in the URL.